### PR TITLE
feat: quick checkout design work

### PR DESCRIPTION
**Pull request by abocsan-plenty**
[AB#99814](https://dev.azure.com/plentymarkets/0af57df9-8662-4901-bb97-8e88b07ff0c9/_workitems/edit/99814)
## Why:

Closes: #ID

## Checklist before requesting a review

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I've updated `docs/changelog/changelog_en.md` and `docs/changelog/changelog_de.md`. If I'm a non-German speaker, I've still updated the file with the English version as a placeholder.

### CODE REVIEW

File: apps/web/nuxt.config.ts [hunk 1]
```diff
@@ -161,6 +161,7 @@ export default defineNuxtConfig({
       useAvif: process.env?.USE_AVIF === '1' ?? false,
       useWebp: process.env?.USE_WEBP === '1' ?? false,
       validateReturnReasons: process.env.VALIDATE_RETURN_REASONS === '1' ?? false,
+      enableQuickCheckoutTimer: true,
```
Comment (maxiroellplenty) [inline apps/web/nuxt.config.ts]: we could already use an env var for that

File: apps/web/components/QuickCheckout/QuickCheckout.vue [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    tag="section"
+    class="h-full md:h-fit m-0 p-0 lg:w-[1000px] overflow-y-auto"
+    aria-labelledby="quick-checkout-modal"
+  >
+    <header>
+      <h2 class="font-bold font-headings text-lg leading-6 md:text-2xl">
+        <span>{{ t('quickCheckout.heading') }}</span>
+      </h2>
+      <div class="absolute right-2 top-2 flex items-center">
+        <span v-if="timer && config.public.enableQuickCheckoutTimer" class="mr-2 text-gray-400">{{ timer }}s</span>
```
Comment (maxiroellplenty) [inline apps/web/components/QuickCheckout/QuickCheckout.vue]: `config.public.enableQuickCheckoutTimer` can we have this info in the composable?

File: apps/web/components/QuickCheckout/QuickCheckout.vue [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        <SfButton square variant="tertiary" @click="close">
+          <SfIconClose />
+        </SfButton>
+      </div>
+    </header>
+
+    <div class="lg:grid lg:grid-cols-2 lg:gap-4">
+      <div class="lg:border-r-2 flex flex-col items-center p-8">
+        <NuxtImg
```
Comment (maxiroellplenty) [inline apps/web/components/QuickCheckout/QuickCheckout.vue:24]: this should use `addModernImageExtension` from `useModernImage` to load modern image formats

File: apps/web/lang/en.json [hunk 4]
```diff
@@ -723,5 +723,12 @@
   "productAttributes": {
     "seeAvailableOptions": "See available options",
     "notValidVariation": "Please select a valid variation."
+  },
+  "quickCheckout": {
+    "heading": "Added to shopping cart",
+    "subTotal": "Items subtotal",
+    "checkYourCart": "Check your cart",
+    "cartContains": "Your cart contains {count} item | Your cart contains {count} items",
```
Comment (maxiroellplenty) [inline apps/web/lang/en.json:743]: how does that work with the `|`

**Review by maxiroellplenty — CHANGES_REQUESTED** [review]

File: apps/web/lang/en.json (hunk 4 above)
Comment (abocsan-plenty) [reply apps/web/lang/en.json:743]: i18n automatically detects pluralization via the pipe character. Left is if it detects 1, right is if it detects > 1.

https://vue-i18n.intlify.dev/guide/essentials/pluralization.html

@maxiroellplenty

**Review by maxiroellplenty — DISMISSED** [review]

**Review by maxiroellplenty — APPROVED** [review]

**Pull request merged**